text/0000-security-advisories.md

```diff
@@ -0,0 +1,259 @@
+- Feature Name: security_advisories
+- Start Date: 2016-08-24
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Crates.io should offer an API to release security advisories for crates. Cargo
+should offer a new command for the same purpose and warn about vulnerable crate
+versions during compilation.
+
+# Motivation
+[motivation]: #motivation
+
+When compared to other ecosystems such as Python's, Rust's broader community
+prefers many single-purpose crates over larger monoliths. This situation,
+together with the strongly encouraged practice of pinning MINOR versions of
+dependencies, slows down the propagation of critical security fixes.
```

Comment:
In the [pre-pre-RFC thread I gave a different motivation](https://internals.rust-lang.org/t/idea-security-advisories-as-part-of-crates-io-metadata/3899). Money quote:

>Unfortunately I'm depending on open source libraries where nobody gets paid for their work. This means that the maintainer of Z is less likely to backport the patch to other MINOR versions, only the latest version gets a PATCH release. The maintainers of X and Y are less likely to update their dependencies either, because they do all of this in their freetime as well. This is a social problem though, and people are just getting started12 writing about this issue.

I agree with the motivation being a bit weak. Do you think the scenario described in that thread above is more realistic?